apis/contour/v1beta1/ingressroute.go

```diff
@@ -50,6 +52,30 @@ type TLS struct {
 	MinimumProtocolVersion string `json:"minimumProtocolVersion,omitempty"`
 }
 
+type HashPolicyHeader struct {
```

Comment:
Each public type and field needs to have a comment so that these types are documented in godoc.